Resources/Prototypes/GameRules/meteorswarms.yml

```
@@ -234,3 +234,6 @@
     - id: ImmovableRodGibstick
       prob: 0.0072
       orGroup: rodProto
+    - id: ImmovableRodWeh
+      prob: 0.0072
```

```suggestion
      prob: 0.00625
```
adjust the probablity so they add up to 1
same for the other rare ones

---

Done!